ocaml/idl/datamodel.ml

```diff
@@ -2537,13 +2537,25 @@ let vm_create_new_blob = call
     ~allowed_roles:_R_VM_POWER_ADMIN
     ()
 
+let vm_bios_string_keys =
+  Enum ("vm_bios_string_keys",
```

Comment:
@robhoes 
On restricting enum keys, API throws unexpected error:
bios = {'bios-vendor': 'abc'} or bios = {'bios-vendor': 'abc', 'enclosure-asset-tag': 'xyz'}
session.xenapi.VM.set_bios_strings(vm, bios)
failswith -> XenAPI.Failure: ['INTERNAL_ERROR', '(Failure "Expected string, got \'[]\'")']

There is no logging for the error - seems like call has not reached message forwarding.
Whereas from CLI - It works fine.